docs/src/dev/future/proposal-transaction-4.asciidoc

```diff
@@ -0,0 +1,113 @@
+////
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+////
+image::apache-tinkerpop-logo.png[width=500,link="https://tinkerpop.apache.org"]
+
+*x.y.z - Proposal 4*
+
+== Tinkerpop native transaction support
```

Comment:
I think the perspective this proposal is coming from is more about "TinkerGraph Transaction Support" than anything general to TinkerPop itself.

Reply:
updated in https://github.com/apache/tinkerpop/pull/2054/commits/4f3f7e916cf4b44db550c67663bd1f4d55bac598